code/datums/gamemode/objectives/target/syndicate/skulls.dm

```diff
@@ -0,0 +1,42 @@
+/datum/objective/target/skulls
+	name = "\[Syndicate\] steal skulls"
+	var/amount
+
+/datum/objective/target/skulls/format_explanation()
+	return "Capture [amount] trophy skulls. They must be from NT employees."
```

Comment:
Might be best to change it to "trophies (decapitated heads)" rather than skulls

Reply:
i was looking at that line myself a while ago, you're probably right